client/components/deposits-information/test/index.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { render } from '@testing-library/react';
+import { merge } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import DepositsInformation from '..';
+import { useAllDeposistsOverviews, useInstantDeposit } from 'data';
+
+jest.mock( 'data', () => ( {
+	useAllDeposistsOverviews: jest.fn(),
+	useInstantDeposit: jest.fn(),
+} ) );
+
+/* eslint-disable camelcase */
+const createMockAccount = ( account = {} ) =>
+	merge(
+		{
+			default_currency: 'eur',
+			deposits_disabled: false,
+			deposits_schedule: {
+				delay_days: 7,
+				interval: 'weekly',
+				weekly_anchor: 'thursday',
+			},
+		},
+		account
+	);
+
+const createMockCurrency = ( currencyCode, extra = {} ) =>
+	merge(
+		{
+			currency: currencyCode,
+			lastPaid: {
+				id: 'po_...',
+				date: 1619395200000,
+				amount: 3160,
+			},
+			nextScheduled: {
+				id: 'wcpay_estimated_weekly_eur_1622678400',
+				date: 1622678400000,
+				amount: 3343,
+			},
+			pending: {
+				amount: 3343,
+				deposits_count: 2,
+			},
+			available: {
+				amount: 2030,
+			},
+		},
+		extra
+	);
+/* eslint-enable camelcase */
+
+const mockOverviews = ( currencies = null, account = null ) => {
+	return useAllDeposistsOverviews.mockReturnValue( {
+		overviews: {
+			currencies: currencies,
+			account: account,
+		},
+		isLoading: null === currencies || ! currencies.length,
+	} );
+};
+
+useInstantDeposit.mockReturnValue( {
+	deposit: undefined,
+	isLoading: false,
+	submit: () => {},
+} );
+
+describe( 'Deposits information', () => {
+	beforeEach( () => {
+		global.wcpaySettings = { zeroDecimalCurrencies: [] };
+	} );
+	afterEach( () => {
+		jest.clearAllMocks();
+	} );
+
+	test( 'renders correctly when loading', () => {
+		mockOverviews();
+		const { container } = render( <DepositsInformation /> );
+		expect( container ).toMatchSnapshot();
+	} );
+
+	test( 'renders correctly with a single currency', () => {
+		mockOverviews( [ createMockCurrency( 'usd' ) ], createMockAccount() );
+
+		const { container } = render( <DepositsInformation /> );
+		expect( container ).toMatchSnapshot();
```

Comment:
Maybe we don't need a structural test for the single currency if we are testing for multiple currencies. Since the structure is repetitive. Test for multiple currency covers for both, let me know if I am missing something.

Reply:
That is right, if there are multiple-currency snapshots, having the simple one is not useful. Removed in https://github.com/Automattic/woocommerce-payments/pull/1892/commits/8b483572ac099caecf6d4c201369efc03b0218bb